### DISCUSSION TITLE
Shortcut to clear mod selection

### DISCUSSION THREAD

**Discussion by farkasma** [ppy/osu#30869] [Ideas]
Title says most of it, in the old client, F1 pulled up the mod selection menu, and F2 could be used to quickly deselect all mods. F1 works in lazer to open and close the mod shelf, but F2 does nothing while there. I guess any button could work, but F2 works for quick use right next to F1, and the visual button layout on the bottom of the screen also lines up.

**Comment by DarkStoorM** [comment]
You can rebind the random song from F2 to F4, lazer uses Backspace for this, but I guess it's more comfortable with one hand 🤔 
My only gripe with this is that you have to disable the mod search focus or get into the habit of pressing `F1 -> Tab -> F2`

**Reply by farkasma** [reply]
I guess in that case F2 would make even more sense, since backspace could still operate on the search field, and there would be no need for tab. In song select, F1/2/3 being the 3 bottom buttons makes sense, and once the mod selection comes up, the two bottom buttons are back and clear. F1 works for back, so F2 would make sense for clearing selection. I guess there is also an argument for old client shortcut parity.
